packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -51,7 +51,7 @@ export class SessionReplayPlugin implements EnrichmentPlugin {
 
   async execute(event: Event) {
     if (event.event_type === DEFAULT_SESSION_START_EVENT && event.session_id) {
-      sessionReplay.setSessionId(event.session_id);
+      sessionReplay.setSessionId({ sessionId: event.session_id });
```

Comment:
should we just keep this as a number instead of object? Doesn't feel like a necessary change right now.

Reply:
Should be number now, was an older commit.